src/testRunner/unittests/tsbuild.ts

```diff
@@ -356,6 +356,19 @@ export class cNew {}`);
                     "TSFILE: /src/tests/index.d.ts",
                 ]);
             });
+
+            it("listEmittedFiles with emitDeclarationOnly", () => {
+                const fs = projFs.shadow();
+                const host = new fakes.SolutionBuilderHost(fs);
+                const builder = createSolutionBuilder(host, ["/src/tests"], { listEmittedFiles: true, declaration: true, emitDeclarationOnly: true });
+                builder.buildAllProjects();

```

Comment:
I think you also need to update this and test case above to include `anotherModule.d.ts.map`, `index.d.ts.map`